capmvm/base-ubuntu/Dockerfile

```diff
@@ -0,0 +1,15 @@
+ARG OS_VERSION
+FROM ubuntu:${OS_VERSION}
+
+RUN apt-get update && apt-get install -y \
+        udev \
+        systemd \
+        systemd-sysv \
+        iproute2 \
+        net-tools \
+        sudo \
+        openssh-server \
```

Comment:
We should probably remove the ssh server from the base image. If users want to allow SSH access they can add it (although for testing we can leave it in temporarily).